test/core/integration/concrete/lockup-base/withdraw/withdraw.tree

```diff
@@ -62,4 +62,5 @@ Withdraw_Integration_Concrete_Test
                                           ├── it should make the withdrawal
                                           ├── it should update the withdrawn amount
                                           ├── it should make Sablier run the recipient hook
+                                          ├── it should transfer the ETH to the lockup contract
```

Comment:
I could not find where its tested in the test code.

Reply:
It is by using withdrawWithBalTest function

Reply:
Same as https://github.com/sablier-labs/v2-core/pull/1087#discussion_r1851757246.